docs/Deploying-PowerAuth-Standard-RESTful-API.md

```diff
@@ -1,20 +1,22 @@
-# Deploying PowerAuth Standard RESTful API Bootstrap
+# Deploying PowerAuth RESTful API
 
 This chapter explains how to deploy PowerAuth Standard RESTful API.
 
-PowerAuth Standard RESTful API is a Java EE application (packaged as an executable WAR file) responsible for exposing the [RESTful API according to the specification](https://github.com/wultra/powerauth-crypto/blob/develop/docs/Standard-RESTful-API.md). It exposes services for end-user applications (PowerAuth Clients), such as the mobile banking app or mobile token app.
+Enrollment Server is a Spring application (packaged as an executable WAR file) responsible for exposing the [RESTful API according to the specification](https://github.com/wultra/powerauth-crypto/blob/develop/docs/Standard-RESTful-API.md). It exposes services for end-user applications (PowerAuth Clients), such as the mobile banking app or mobile token app.
 
 You can use this application in case you need to use PowerAuth and cannot integrate it in your own API using our integration libraries.
 
-## Downloading PowerAuth Standard RESTful API
+## Downloading Enrollment Server
 
-You can download the latest `powerauth-restful-server.war` at the releases page:
+You can download the latest `enrollment-server.war` at the releases page:
 
-- https://github.com/wultra/powerauth-restful-integration/releases
+- https://github.com/wultra/enrollment-server/releases
 
-## Configuring PowerAuth Standard RESTful API
+_Note: The enrollment server component will need to be customized in case you need to customize the activation process. The customization is described in the tutorial [Implementing the Server-Side for Authentication in Mobile Banking Apps (SCA)](https://developers.wultra.com/products/mobile-token/2021-05/tutorials/Authentication-in-Mobile-Apps/Server-Side-Tutorial#deploying-the-enrollment-server)._
```

Comment:
Use the DocuCheck metadata here:

```xml
<-- begin box warning -->
<-- end -->
```

Let's edit this across this documention.

Reply:
Ok, I will add it.